application/src/main/webapp/vue-app/components/ExoChatApp.vue

```diff
@@ -431,6 +432,10 @@ export default {
       } else {
         return $('#PlatformAdminToolbarContainer').css('display', 'block') && $('#UITopBarContainerParent').css('display', 'block');
       }
+    },
+    removeClassPortal() {
```

Comment:
This seems to be more a hack, could you find a better way to remove the bottom blank space ?